data/human/jobs.txt

```diff
@@ -2737,6 +2737,251 @@ mission "Bounty Hunting (Small, Boarding, Entering)"
 		payment 175000
 		dialog "The government of <planet> gratefully pays you <payment> for apprehending the criminal."
 
+mission "Bounty Hunting (Small, Hidden)"
+	name "Disguised bandit near <system>"
+	description "A low-end pirate warship named the <npc> has masqueraded as a merchant ship near the <system> system to avoid detection. Select merchant ships until you find a ship with a matching name, then destroy it and return to <planet> for payment (<payment>)."
+	repeat
+	job
+	to offer
+		"combat rating" > 300
+		random < 20
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	npc kill
+		government "Bounty (Disguised)"
+		personality heroic staying
+		system
+			distance 1 2
+		fleet
+			names "pirate"
+			variant
+				"Firebird"
+			variant
+				"Firebird (Missile)"
+			variant
+				"Splinter"
+			variant
+				"Splinter (Laser)"
+			variant
+				"Osprey"
+			variant
+				"Osprey (Laser)"
+			variant
+				"Bastion"
+			variant
+				"Manta"
+			variant
+				"Manta (Proton)"
+		dialog phrase "generic hunted bounty eliminated dialog"
+	on visit
+		dialog phrase "generic bounty hunting on visit"
+	on complete
+		payment 250000
+		dialog phrase "generic bounty hunting payment dialog"
+
+mission "Bounty Hunting (Medium, Hidden)"
+	name "Disguised pirate near <system>"
+	description "A medium pirate warship named the <npc> has masqueraded as a merchant ship near the <system> system to avoid detection. Select merchant ships until you find a ship with a matching name, then destroy it and return to <planet> for payment (<payment>)."
+	repeat
+	job
+	to offer
+		"combat rating" > 400
+		random < 10
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	npc kill
+		government "Bounty (Disguised)"
+		personality heroic staying
+		system
+			distance 1 2
+		fleet
+			names "pirate"
+			variant
+				"Protector"
+			variant
+				"Protector (Laser)"
+			variant
+				"Osprey (Missile)"
+			variant
+				"Bastion (Heavy)"
+			variant
+				"Bastion (Laser)"
+			variant
+				"Firebird (Plasma)"
+		dialog phrase "generic hunted bounty eliminated dialog"
+	on visit
+		dialog phrase "generic bounty hunting on visit"
+	on complete
+		payment 350000
+		dialog phrase "generic bounty hunting payment dialog"
+
+mission "Bounty Hunting (Big, Hidden)"
+	name "Disguised warlord near <system>"
+	description "A heavy warship known as the <npc> has masqueraded as a merchant ship near the <system> system to avoid detection. Select merchant ships until you find a ship with a matching name, then destroy it and return to <planet> for payment (<payment>)."
```

Comment:
```suggestion
	description "A heavy warship known as the <npc> is masquerading as a merchant ship near the <system> system to avoid detection. Select merchant ships until you find a ship with a matching name, then destroy it and return to <planet> for payment (<payment>)."
```